src/modules/groups/groups.js

```diff
@@ -0,0 +1,87 @@
+const express = require('express');
+const axios = require('axios');
+const { validateMiddleware } = require('../../middlewares');
+const { groupDto } = require('./dtos');
+
+const router = express.Router();
+
+const getAccessToken = async () => {
+  const { CLIENT_SECRET, CLIENT_ID, DEFAULT_MS_GRAPH_SCOPE, MS_LOGIN_HOST_NAME, TENANT_ID } =
+    process.env;
+  try {
+    const URL = `${MS_LOGIN_HOST_NAME}/${TENANT_ID}/oauth2/v2.0/token`;
+    const { data } = await axios.post(
+      URL,
+      {
+        grant_type: 'client_credentials',
+        client_secret: CLIENT_SECRET,
+        client_id: CLIENT_ID,
+        scope: DEFAULT_MS_GRAPH_SCOPE
+      },
+      {
+        headers: {
+          'Content-Type': 'multipart/form-data'
+        }
+      }
+    );
+    return data?.access_token ? `Bearer ${data?.access_token}` : undefined;
+  } catch (error) {
+    return null;
+  }
+};
+
+router.get('/token', async (req, res) => {
+  const token = (await getAccessToken()) || {};
+  if (!token) {
```

Comment:
it will never reach this since u are initializing it with {}

Reply:
Resolved in 5cc9d909d4bb497c5c867ab54fdb7d2384701dc5